pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1105,15 +1112,15 @@ func TestPodTimestamp(t *testing.T) {
 		{
 			name: "add one pod to BackoffQ and move it to activeQ",
 			operations: []operation{
-				addPodActiveQ(pInfo2), addPodBackoffQ(pInfo1), flushBackoffQ(), moveAllToActiveQ(),
+				addPodActiveQ(pInfo2), addPodBackoffQ(pInfo1), backoffPod(pInfo1), flushBackoffQ(), moveAllToActiveQ(),
 			},
 			expected: []*podInfo{pInfo1, pInfo2},
 		},
 	}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			queue = NewPriorityQueue(nil)
+			queue = NewPriorityQueueWithClock(nil, clock.NewFakeClock(time.Now()))
```

Comment:
time.Now() changes every time you read it. In order to have a robust test, please read it once in the beginning of the function and use that time here and for initializing `pInfo1` and `pInfo2` in lines 1026 and 1030.

Reply:
Yes, Thanks for your advice. Done